File: Module-CP/Assignments/assignment14/Pass_by_address(array)/q1.c
#include<stdio.h>
typedef struct Student{
   int rollNo;
   char name[20];
   int marks;
}Student;
void storeStudent(Student*,int);
void displayStud(Student*,int);
int main(){
    Student arr[5];

    storeStudent(arr,5);
    displayStud(arr,5);
}
void storeStudent(Student*arr,int size){
    printf("Enter Details of Student:");
    for(int i=0;i<size;i++){
        printf("Enter roll_no:");
        scanf("%d",&arr[i].rollNo);
        printf("Enter name:");
        scanf("%s",arr[i].name);
        printf("Enter marks:");
        scanf("%d",&arr[i].marks);
    }
}
void displayStud(Student*arr,int size){
      printf("-----Student Details-----");
        for(int i=0;i<size;i++){
        printf("\nRollNo = %d",arr[i].rollNo);
        printf("\nName = %s",arr[i].name);
        printf("\nMarks = %d",arr[i].marks);
    }
}

File: Module-Core Java/Assignments/C to Java/Assignment 14/Main/q1.java
import java.util.*;
class Student{
    int rollNo;
    String name;
    int marks;
}
public class q1{
    public static void main(String[] args) {
        Student s1=new Student();
        Student s2=new Student();

        Scanner sc = new Scanner(System.in);
        
        System.out.println("Student Name:");
        s1.name=sc.next();
        System.out.println("Student RollNo:");
        s1.rollNo=sc.nextInt();
        System.out.println("Student Marks:");
        s1.marks=sc.nextInt();
       
        System.out.println("Student Name:");
        s2.name=sc.next();
        System.out.println("Student RollNo:");
        s2.rollNo=sc.nextInt();
        System.out.println("Student Marks:");
        s2.marks=sc.nextInt();
        
        System.out.println("Student Name: "+s1.name);
        System.out.println("Student RollNO: "+s1.rollNo);
        System.out.println("Student marks: "+s1.marks);
        System.out.println("Student Name: "+s2.name);
        System.out.println("Student RollNO: "+s2.rollNo);
        System.out.println("Student marks: "+s2.marks);

    }
}